patches/jsc.patch

```diff
@@ -86,7 +86,7 @@ diff -aur target-org/webkit/Source/cmake/OptionsJSCOnly.cmake target/webkit/Sour
 -endif ()
 +set(ENABLE_API_TESTS OFF)
 +set(ENABLE_SAMPLING_PROFILER OFF)
-+set(ENABLE_DFG_JIT OFF)
++set(ENABLE_DFG_JIT ON)
```

Comment:
It's still possible that there is some negative performance impact from DFG, performance improvement I observed is relative to 32-bit, so it is possible that it was caused not by dfg but by platform specific code(e.g. no need to emulate 64-bit values, 64-bit values become atomic, some locking is not necessary).

I left DFG enabled because compilation was failing without it. It's probably fixable, but I think it's lower priority than fixing this crash.